packages/seducer/src/logger.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable no-use-before-define */
+/**
+ * Summary: Prints the current processing action via the dev console, with it's previous state and the next state after passing inside the reducer.
+ *
+ * @param {{unknown}}           prevState           The current state of the store
+ * @param {{unknown}}           nextState           The next state after being pass through the reducer
```

Comment:
and why they're `{{unknown}}` not `{unknow}`?

Reply:
because I'm ESL lol 🤣

Reply:
sorry I was going to type `{unknown}` 😅